engine/common/mod_bmodel.c

```diff
@@ -430,13 +430,28 @@ void Mod_PrintWorldStats_f( void )
 
 	Con_Printf( "=== Total BSP file data space used: %s ===\n", Q_memprint( totalmemory ));
 	Con_Printf( "World size ( %g %g %g ) units\n", world.size[0], world.size[1], world.size[2] );
-	Con_Printf( "Supports transparency world water: %s\n", FBitSet( world.flags, FWORLD_WATERALPHA ) ? "Yes" : "No" );
-	Con_Printf( "Lighting: %s\n", FBitSet( w->flags, MODEL_COLORED_LIGHTING ) ? "colored" : "monochrome" );
+	if( FBitSet( world.flags, FWORLD_WATERALPHA ))
+		Con_Printf( "Supports transparency world water: Yes\n" );
+	else
+		Con_Printf( "Supports transparency world water: No\n" );
+	if( FBitSet( w->flags, MODEL_COLORED_LIGHTING ))
+		Con_Printf( "Lighting: colored\n" );
+	else
+		Con_Printf( "Lighting: monochrome\n" );
 	Con_Printf( "World total leafs: %d\n", worldmodel->numleafs + 1 );
 	Con_Printf( "original name: ^1%s\n", worldmodel->name );
-	Con_Printf( "internal name: %s\n", (world.message[0]) ? va( "^2%s", world.message ) : "none" );
-	Con_Printf( "map compiler: %s\n", (world.compiler[0]) ? va( "^3%s", world.compiler ) : "unknown" );
-	Con_Printf( "map editor: %s\n", (world.generator[0]) ? va( "^2%s", world.generator ) : "unknown" );
+	if( world.message[0] )
+		Con_Printf( "internal name: ^2%s\n", world.message );
+	else
+		Con_Printf( "internal name: none\n" );
+	if( world.compiler[0] )
+		Con_Printf( "map compiler: ^3%s\n", world.compiler );
+	else
+		Con_Printf( "map compiler: unknown\n" );
+	if( world.generator[0] )
+		Con_Printf( "map editor: ^2%s\n", world.generator );
+	else
+		Con_Printf( "map editor: unknown\n" );
```

Comment:
It's disgusting. Can you stop wasting your time on this? Leave it as ternary operation, it's ok.

Reply:
If you're worried about coloring, we can color "unknown" too for the sake of simplicity and getting rid of va()